crypto/sleep.c

```diff
@@ -46,27 +46,15 @@ void OSSL_sleep(uint64_t millis)
 {
     /*
      * Windows' Sleep() takes a DWORD argument, which is smaller than
-     * a uint64_t, so we need to split the two to shut the compiler up.
+     * a uint64_t, so we need to limit it to 49 days, which should be enough.
```

Comment:
This comment, saying "we might not actually wait as long as you ask for, but the maximum is still a long time" should really be in the function documentation, not implementation

Reply:
Good point.